hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeFactoryAuthProxy.java

```diff
@@ -57,10 +46,16 @@
 import com.baidu.hugegraph.traversal.optimize.HugeCountStepStrategy;
 import com.baidu.hugegraph.traversal.optimize.HugeGraphStepStrategy;
 import com.baidu.hugegraph.traversal.optimize.HugeVertexStepStrategy;
+import com.baidu.hugegraph.util.ReflectionProxy;
 import com.baidu.hugegraph.variables.HugeVariables;
 import com.google.common.collect.ImmutableSet;
+import org.apache.commons.configuration.Configuration;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.*;
```

Comment:
don't import "*"

Reply:
ok, this '*' is auto-created by idea tool, I change it;